tensorflow/python/keras/applications/mobilenet_v2.py

```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
     ValueError: in case of invalid argument for `weights`,
       or invalid input shape or invalid alpha, rows when
       weights='imagenet'
+      
+  Usage Example:
+  >>> from tensorflow.keras.applications import MobileNetV2
+  >>> IMG_SHAPE = (224, 224, 3)
```

Comment:
Better to create a small fake image (using `tf.constant`) and run that through the model

Reply:
If I created a fake image and passed it to model, wouldn't it give a large array as output?